bedrock/firefox/tests/test_views.py

```diff
@@ -634,6 +634,39 @@ def test_firefox_home(self, render_mock):
         assert template == ["firefox/home/index-master.html"]
 
 
+class TestFirefoxGA(TestCase):
+    def assert_ga_attr(self, response):
+        cta_types = ["link", "button"]
```

Comment:
This looks to only be used in the `if cta_data` below. Similar to how you wrote the `cta_link` section, this could be moved within the `if` block. Or even inline with the `any` call since/if it is only used once.